src/client/cmd/launch.cpp

```diff
@@ -199,7 +199,11 @@ mp::ReturnCode cmd::Launch::request_launch()
     auto on_success = [this, &spinner](mp::LaunchReply& reply) {
         spinner.stop();
 
-        if (reply.metrics_pending())
+        if (term->is_live() && update_available(reply.update_info()))
```

Comment:
So if terminal isn't live, we "lose" the update notification until the "between notices" timer passes?

Reply:
Yep, as daemon has no idea if client's terminal is live or not. I considered adding way for client to say "I can show update message", that daemon can then do the right thing, but wasn't sure if it was worth the trouble.

Reply:
Right, that's what I was thinking. If people use it through, say, `snapcraft` a lot, it could never show when using interactively.

Reply:
todo added